scanapi/config_loader.py

```diff
@@ -1,6 +1,4 @@
-"""
-Code based on solution https://gist.github.com/joshbode/569627ced3076931b02f
-"""
+"""Code based on solution https://gist.github.com/joshbode/569627ced3076931b02f"""
```

Comment:
I think DeepSource has a false positive with links because [it marked this docstring as an occurrence](https://deepsource.io/gh/scanapi/scanapi/issue/FLK-D200/occurrences
) and now it says [it's too long](https://deepsource.io/gh/scanapi/scanapi/run/9de3b112-05d2-4912-a87b-b025d7343618/python/FLK-W505).

The same happens with https://github.com/scanapi/scanapi/blob/35ef8d979796cb3835bcbeb0c09eac7f4f0d63ea/scanapi/exit_code.py#L1-L4

Reply:
>now it says it's too long.

@marcorichetta for the line too long. ScanAPI has a set max length of a string is `80` and when you convert it into one line it's 82 chars. That's why the error `doc line too long (82 > 80 characters)` 

If you are trying to put it in a single line maybe you can try to update the docstring text something like that might have the chars less than or up to 80. 

```
"""Code Refrence https://gist.github.com/joshbode/569627ced3076931b02f"""
```

Reply:
Thanks @Pradhvan. I fixed it with your suggestion and now it passed the checks.